src/Cli.Tests/TestLoggerSupport.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Cli.Tests;
+
+internal static class TestLoggerSupport
+{
+    public static ILoggerFactory ProvisionLoggerFactory() =>
```

Comment:
Maybe we could move it to the TestHelper class in Cli.Tests ?

Reply:
I noted in the PR description that it was a deliberate decision to not put it in that class. When doing the changes for #1402 I had many conflicts on these "helper" classes as they can be iterated on frequently. This isn't just isolated to test projects either.

So as a result I'll always try to introduce a new class which is specific for the "feature", even if it only has a single method in it, as it become clearer where code will exist when scanning through a repo.